chromium_src/chrome/browser/unexpire_flags.cc

```diff
@@ -35,7 +35,7 @@ bool IsFlagExpired(const flags_ui::FlagsStorage* storage,
     return true;
   }
 #endif  // BUILDFLAG(ENABLE_PLAYLIST) && BUILDFLAG(IS_ANDROID)
-#if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD)
+#if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD) && BUILDFLAG(IS_WIN)
```

Comment:
`kBraveVPNWireguardFeatureInternalName ` still needed for correct migration on win from the feature flag to the pref

Reply:
Yeah, I thought we can remove it from about_flags.cc while keeping the flag itself. I might be wrong, removing it from the list in about_flags.cc only hides it from `brave://flags` and we'll have access to the feature flag from code.

Reply:
AFAIK, feature flag gives default value if it's removed from `brave://flags`.
So, @spylogsster keeps it from there untill migration done.

Reply:
yes, thanks @simonhong, `about_flags.cc` patches add it to `kFeatureEntries` otherwise, if there is no flag, it will be reset to defaul state all time ignoring saved value.

Reply:
Thanks, didn't know that 👍